onnxsim/__main__.py

```diff
@@ -18,26 +19,49 @@ def main():
     parser.add_argument('--skip-optimization', help='Skip optimization of ONNX optimizers.',
                         action='store_true')
     parser.add_argument(
-        '--input-shape', help='The manually-set static input shape, useful when the input shape is dynamic. The value should be "input_name:dim0,dim1,...,dimN" or simply "dim0,dim1,...,dimN" when there is only one input, for example, "data:1,3,224,224" or "1,3,224,224". Note: you might want to use some visualization tools like netron to make sure what the input name and dimension ordering (NCHW or NHWC) is.', type=str, nargs='+')
+        '--input-shape', help='The manually-set static input shape, useful when the input shape is dynamic. The value should be "input_name1:dim0,dim1,...,dimN input_name2:dim0,dim1...dimN", for example, "data1:1,3,224,224 data2:1,3,224,224". Note: you might want to use some visualization tools like netron to make sure what the input name and dimension ordering (NCHW or NHWC) is.', type=str, nargs='+')
     parser.add_argument(
         '--skip-optimizer', help='Skip a certain ONNX optimizer', type=str, nargs='+')
     parser.add_argument('--skip-shape-inference',
                         help='Skip shape inference. Shape inference causes segfault on some large models', action='store_true')
+    parser.add_argument(
+        '--input_data', help='input data, The value should be "input_name1:xxx1.bin  input_name2:xxx2.bin ...", input data should be a binary data file.', type=str, nargs='+')
+    parser.add_argument(
+        '--custom_lib', help="custom lib, if you have custom onnxruntime backend you should use this to register you custom op", type=str)
     args = parser.parse_args()
     print("Simplifying...")
-    input_shapes = {}
+    input_shapes = dict()
```

Comment:
Why use `dict()` instead of `{}`? :)

Reply:
{} is confused, if it is a set or a dict, so i used dict()